wiki.md

```diff
@@ -128,9 +128,55 @@ then go to [Local Demo Website](http://127.0.0.1:8000/) to see an online visuali
     ```commandline
     docker cp container_id:/path/in/container /path/on/host
     ```
+
+### Packaging Your Software into a Docker Image
+- Build the image:
+    ```commandline
+    docker build --build-arg "DEFAULT_CMD=python3 online_log/app.py" -t chatdev:latest .
+    ```
+    Make sure you replace:
+    - `online_log/app.py` with the relative path to your Python application's entry point.
+    - `chatdev` with a name that you want to give to your Docker image.
+
+- Test the image:
+    ```commandline
+    docker run -itp 8000:8000 chatdev:latest
+    ```
+    Ensure you replace `chatdev` with the name you used when building the image.
+
 ### Official Docker Image
 - in preparation
 
+## Start the Application with Docker Compose
+
+To run your ChatDev application with Docker Compose, simply:
+
+1. Find the `.env.sample` file in the root directory of your ChatDev project.
+2. Copy this file and rename it to `.env`.
+3. Open the `.env` file with the text editor of your choice.
+4. Replace `<Your OpenAI API Key>` with your actual OpenAI API key from OpenAI. Add other environment variables for local development as needed.
+5. Initiate the application by running:
```

Comment:
What is the point of the .env.sample? just leave it as .env

Reply:
good point